webpack.config.js

```diff
@@ -20,6 +20,9 @@ module.exports = env => ({
             path.resolve(__dirname, 'node_modules/leaflet/dist/leaflet.css'),
             './ts/members-map.ts',
         ],
+        membershipGraphs: [
```

Comment:
if this entrypoint has only one file, it can look like the ones below, that don't need an array